src/app/modules/group/components/group-invite-users/group-invite-users.component.ts

```diff
@@ -1,24 +1,134 @@
 
-  constructor() { }
+  messages: Message[] = [];
+
+  constructor(
+    private createGroupInvitationsService: CreateGroupInvitationsService,
+    private messageService: MessageService,
+    ) {}
 
   ngOnInit(): void {}
 
-  onInvite() {
-    this.showMsg = !this.showMsg;
+  private processRequestError(_err: any) {
+    this.messageService.add({
+      severity: 'error',
+      summary: 'Error',
+      detail: ERROR_MESSAGE.fail,
+      life: TOAST_LENGTH,
+    });
+  }
+
+  private displayResponse(response: Map<string, InvitationResult>) {
+
+    const sucessInvites: string[] = Array.from(response.entries()).filter((e) => e[1] == InvitationResult.Success).map((e) => e[0]);
+    const alreadyInvited: string[] = Array.from(response.entries()).filter((e) => e[1] == InvitationResult.AlreadyInvited).map((e) => e[0]);
+    const notFoundUsers: string[] = Array.from(response.entries()).filter((e) => e[1] == InvitationResult.NotFound).map((e) => e[0]);
+    const invalidInvites: string[] = Array.from(response.entries()).filter((e) => e[1] == InvitationResult.Error).map((e) => e[0]);
+
+    if (sucessInvites.length > 0)
+      this.messages.push({
+        type: 'success',
+        summary: `${sucessInvites.length} user(s) invited successfully: `,
+        detail: `${sucessInvites.join(', ')}`,
+      });
+
+    if (alreadyInvited.length > 0)
+      this.messages.push({
+        type: 'info',
+        summary: `${alreadyInvited.length} user(s) have already been invited: `,
+        detail: `${alreadyInvited.join(', ')}`,
+      });
+
+    if (notFoundUsers.length > 0)
+      this.messages.push({
+        type: 'error',
+        summary: `${notFoundUsers.length} user login(s) not found: `,
+        detail: `${notFoundUsers.join(', ')}`,
+      });
+
+    if (invalidInvites.length > 0)
+      this.messages.push({
+        type: 'error',
+        summary: `${invalidInvites.length} user login(s) could not be invited: `,
+        detail: `${invalidInvites.join(', ')}`,
+      });
+  }
+
+  /* events */
+
+  onTextChange(text:string)
+  {
+    this.state = 'ready';
+
+    const logins = text.split(',').filter(login => login.length > 0);
+
+    if (logins.length == 0) {
+      this.state = 'empty';
+    } else if (logins.length >= 100) {
+      this.state = 'too_many';
+    }
   }
 
+  onInviteClicked() {
+    // clear the messages
+    this.messages = [];
+
+    // remove empty logins and duplicates
+    const logins = this.textArea.value.split(',')
+      .map((login) => login.trim())
+      .filter(function (login, index, self) {
+        return self.indexOf(login) === index && login !== '';
+      });
+
+    if (logins.length == 0) return;
+
+    // disable UI
+    this.state = 'loading';
+
+    this.createGroupInvitationsService.createInvitations(this.group.id, logins).subscribe(
+      (res) => {
+        this.displayResponse(res);
+
+        // Clear the textarea
+        if (this.textArea)
+          this.textArea.setValue('');
```

Comment:
on a single line
(even it is not enforced by the linter here, I prefer only the 2 following options: without bracket on a single line or with brackets on multiple lines.)